tests/acceptance/features/apiSharingNg1/sharedWithMe.feature

```diff
@@ -5268,3 +5268,113 @@ Feature: an user gets the resources shared to them
     Then the HTTP status code should be "200"
     And user "Brian" should not have a share "FolderToShare" shared by user "Alice" from space "NewSpace"
     And user "Brian" should not be able to download file "FolderToShare/lorem.txt" from space "Shares"
+
+  @env-config
+  Scenario Outline: check share access after updating the permission role of a shared folder from other roles to Denied (Personal Space)
+    Given using spaces DAV path
+    And the administrator has enabled the permissions role "Denied"
+    And user "Alice" has created folder "FolderToShare"
+    And user "Alice" has uploaded file with content "hello world" to "FolderToShare/lorem.txt"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | FolderToShare      |
+      | space           | Personal           |
+      | sharee          | Brian              |
+      | shareType       | user               |
+      | permissionsRole | <permissions-role> |
+    And user "Alice" has updated the last resource share with the following properties:
+      | permissionsRole | Denied        |
+      | space           | Personal      |
+      | resource        | FolderToShare |
+    When user "Brian" lists the shares shared with him using the Graph API
+    Then the HTTP status code should be "200"
+    And user "Brian" should not have a share "FolderToShare" shared by user "Alice" from space "Personal"
+    And user "Brian" should not be able to download file "FolderToShare/lorem.txt" from space "Shares"
+    Examples:
+      | permissions-role |
+      | Editor           |
+      | Viewer           |
+      | Uploader         |
+
+  @env-config
+  Scenario Outline: check share access after updating the permission role of a shared folder from Denied to other roles (Personal Space)
+    Given using spaces DAV path
+    And the administrator has enabled the permissions role "Denied"
+    And user "Alice" has created folder "FolderToShare"
+    And user "Alice" has uploaded file with content "hello world" to "FolderToShare/lorem.txt"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | FolderToShare |
+      | space           | Personal      |
+      | sharee          | Brian         |
+      | shareType       | user          |
+      | permissionsRole | Denied        |
+    And user "Alice" has updated the last resource share with the following properties:
+      | permissionsRole | <permissions-role> |
+      | space           | Personal           |
+      | resource        | FolderToShare      |
+    When user "Brian" lists the shares shared with him using the Graph API
+    Then the HTTP status code should be "200"
+    And user "Brian" should have a share "FolderToShare" shared by user "Alice" from space "Personal"
+    And user "Brian" should be able to download file "FolderToShare/lorem.txt" from space "Shares"
```

Comment:
i think we can remove this step 
```suggestion
```

Reply:
let's keep it.

Reply:
> let's keep it. so that the test can confirm some actions are possible after role change

I read it wrong, the content check step below also checks the download. so, we can remove this download step

Reply:
Oh okeii. I didn't know that happens. :slightly_smiling_face: